src/XRP/issued-currency-client.ts

```diff
@@ -479,6 +479,32 @@ export default class IssuedCurrencyClient {
     )
   }
 
+  /**
+   * Unfreezes the trust line between this account (issuing account) and another account.
+   * Note that the trust line's limit is set to 0.
+   *
+   * @see https://xrpl.org/freezes.html#enabling-or-disabling-individual-freeze
+   *
+   * @param accountToUnfreeze The X-Address of the account which to unfreeze a trust line.
```

Comment:
same note here about this confusing name...  I'd also call this `trustlineAccount` and the second parameter `trustlineCurrencyName` or just `trustlineCurrency` ...

Reply:
got it, per our conversation off line i changed this to `trustLinePeerAccount`